### ISSUE TITLE
Disabled item within scrollable table still responds to keyboard input

### ISSUE THREAD

**Issue by lethal-guitar** [ocornut/imgui#5189]
**Version/Branch of Dear ImGui:**

Version: 1.88 WIP (18717)
Branch: `master`

**Back-end/Renderer/Compiler/OS**

Back-ends: imgui_impl_sdl.cpp + imgui_impl_opengl3.cpp
Operating System: Windows 10, Mac OS 10.14

**My Issue/Question:**

When using a scrollable table, `BeginDisabled`/`EndDisabled` within a table cell doesn't prevent keyboard input (`ImGuiConfigFlags_NavEnableKeyboard`). This only seems to be a problem for the first row.
I experienced this with a checkbox. I haven't checked other widgets, and I haven't tested gamepad input yet.

Mouse interaction is prevented as expected, and this issue does not occur if `ImGuiTableFlags_ScrollY` is not used.

**Screenshots/Video**

https://user-images.githubusercontent.com/1821027/163004112-d6e93460-c3ab-418f-849a-c340be87d4ed.mov

**Standalone, minimal, complete and verifiable example:** _(see https://github.com/ocornut/imgui/issues/2261)_
```
if (ImGui::BeginTable("example", 1, ImGuiTableFlags_ScrollY))
{
  ImGui::TableNextRow();
  ImGui::TableNextColumn();

  ImGui::BeginDisabled(true);
  ImGui::Checkbox("Test", &test);
  ImGui::EndDisabled();

  ImGui::EndTable();
}
```

**Comment by ocornut** [comment]
Thank for reporting! Fixed with 088487a3d44dc75db73ed4aff9bbd224604102ac.

As a understand, the issue doesn't have to do with tables or "first row", but it happened when starting navigation inside a window where there are ONLY disabled items. Should be fixed now.

**Comment by lethal-guitar** [comment]
Ah nice, that was very quick! Thank you.

> As a understand, the issue doesn't have to do with tables or "first row", but it happened when starting navigation inside a window where there are ONLY disabled items.

Ah yeah that could be, in my real world example the table had more than one row but all checkboxes were disabled.
